src/tradelocker/tradelocker_api.py

```diff
@@ -1341,3 +1375,156 @@ def modify_order(
+        return response_status == "ok"
+
+
+    @log_func
+    @tl_typechecked
+    # TODO: this should probably be further expanded / validated
+    def get_orders(self) -> bool:
+        """Modifies a open position.
+
+        Args:
+            order_id (int): Order Id
+            modification_params (_ModificationParamsType): Order modification details
+
+        Returns:
+            bool: True on success, False on error
+        """
+        print('AccessToken: ', self.get_access_token())
+        route_url = f"{self._base_url}/trade/accounts/{self.account_id}/orders"
+
+        self.log.info(f"Getting Orders")
+
+        response = requests.get(
+            url=route_url,
+            headers=self._get_headers({"Content-type": "application/json"}),
+            timeout=_TIMEOUT,
+        )
+        response_json = self._get_response_json(response)
+        self.log.info(f"Order modification response: {response_json}")
+        response_status: str = get_nested_key(response_json, ["s"], str)
+        return response_status == "ok"
+    
+    @log_func
+    @tl_typechecked
+    # TODO: this should probably be further expanded / validated
+    def get_positions(self) -> bool:
+        """Modifies a open position.
+
+        Args:
+            order_id (int): Order Id
+            modification_params (_ModificationParamsType): Order modification details
+
+        Returns:
+            bool: True on success, False on error
+        """
+        print('AccessToken: ', self.get_access_token())
+        route_url = f"{self._base_url}/trade/accounts/{self.account_id}/positions"
+
+        self.log.info(f"Getting positions")
+
+        response = requests.get(
+            url=route_url,
+            headers=self._get_headers({"Content-type": "application/json"}),
+            timeout=_TIMEOUT,
+        )
+        response_json = self._get_response_json(response)
+        self.log.info(f"Position modification response: {response_json}")
+        response_status: str = get_nested_key(response_json, ["s"], str)
+        return response_status == "ok"
+    
+    @log_func
+    @tl_typechecked
+    # TODO: this should probably be further expanded / validated
+    def get_executions(self) -> bool:
+        """Retrieves executions.
+
+        Returns:
+            bool: True on success, False on error
+        """
+        route_url = f"{self._base_url}/trade/accounts/{self.account_id}/executions"
+
+        # self.log.info(f"Getting executions")
+
+        response = requests.get(
+            url=route_url,
+            headers=self._get_headers({"Content-type": "application/json"}),
+            timeout=_TIMEOUT,
+        )
+        response_json = self._get_response_json(response)
+        self.log.info(f"Executions response: {response_json}")
+        response_status: str = get_nested_key(response_json, ["s"], str)
+        return response_status == "ok"
+    
+    @log_func
+    @tl_typechecked
+    def get_orders_history(self):
```

Comment:
Please remove this whole function -- there is already a function self.get_all_orders(history=True)